module2/gift-certificate-system/persistence/src/main/java/com/epam/esm/dao/HibernateAnnotationUtil.java

```diff
@@ -0,0 +1,39 @@
+package com.epam.esm.dao;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.Metadata;
+import org.hibernate.boot.MetadataSources;
+import org.hibernate.boot.registry.StandardServiceRegistry;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.service.ServiceRegistry;
+
+public class HibernateAnnotationUtil {
```

Comment:
Check Utility class implementation in Lombok. https://projectlombok.org/features/experimental/UtilityClass